.github/workflows/env-setup-delete-redis.yml

```diff
@@ -0,0 +1,34 @@
+name: Env Setup - Delete Redis in Openshift
+run-name: Env Setup - Delete Redis in Openshift from ${{ inputs.environment }} using ${{ inputs.gitRef }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Environment"
+        required: true
+        type: environment
+      gitRef:
```

Comment:
We have been willing to remove this from the input and use it directly from the reference tag selected to execute the workflow. I believe that `github.ref_name` would be the one ([source reference](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context)).
Can we give a try?
Instead of
```yml
 - name: Checkout Target Branch
        uses: actions/checkout@v4
        with:
          ref: ${{ inputs.gitRef }}
```
it would be 
```yml
 - name: Checkout Target Branch
        uses: actions/checkout@v4
        with:
          ref: ${{ github.ref_name }}
```

Reply:
If the suggestion was accepted, please remove the input parameter.
Same for the other files.